ooniprobe/Utility/ExceptionUtility.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+#import <Crashlytics/Crashlytics.h>
```

Comment:
I believe you should import this header inside the implementation, not inside the interface header

Reply:
done